src/enums/VentilationBreathingType.ts

```diff
@@ -8,7 +8,7 @@ export class VentilationBreathingTypeUtils {
     breathingType: VentilationBreathingType,
   ): string {
     if (breathingType === VentilationBreathingType.Assisted) {
-      return 'AC-';
+      return 'AC-V';
```

Comment:
you can remove this, I will refactor it afterwards

Reply:
okay